offworld_gym/__init__.py

```diff
@@ -48,25 +48,25 @@
     # OffWorld Monolith Simulated replica with discrete actions
     register(
         id='OffWorldDockerMonolithDiscreteSim-v0',
-        entry_point='offworld_gym.envs.gazebo_docker:OffWorldDockerMonolithDiscreteEnv'
+        entry_point='offworld_gym.envs.gazebo:OffWorldDockerMonolithDiscreteEnv'
     )
 
     # OffWorld Monolith Simulated replica with continous actions
     register(
         id='OffWorldDockerMonolithContinuousSim-v0',
-        entry_point='offworld_gym.envs.gazebo_docker:OffWorldDockerMonolithContinuousEnv'
+        entry_point='offworld_gym.envs.gazebo:OffWorldDockerMonolithContinuousEnv'
     )
 
-    # OffWorld Monolith Simulated replica with discrete actions
-    register(
-        id='OffWorldDockerMonolithObstacleDiscreteSim-v0',
-        entry_point='offworld_gym.envs.gazebo_docker:OffWorldDockerMonolithObstacleDiscreteEnv'
-    )
+    # # OffWorld Monolith Simulated replica with discrete actions
```

Comment:
Please delete the following commented statements

Reply:
done